contents/handbook/people/compensation.md

```diff
@@ -56,8 +64,10 @@ If this represents an increase in pay, _we need to approve this change in advanc
 If you plan on spending >4 months/year in a place different from your home base, that will be adjusted based on your location after 4 months.
 
 For a trip with many destinations over a period of more than 4 months, the location factor will be located along the various places you intend to stay, averaged by the amount of time spent.
+
+If you are uncertain about where you're travelling to in advance, and you are travelling over 4 months, then we will keep your pay the same as when you departed. 4 months later, we will make a manual adjustment to edit your next pay amount based on the average of the previous 4 months. This will take place every 4 months until you are done travelling. If the adjustment would require reclaiming more than your entire pay amount (for example if you moved from one of the world's most expensive areasa to one of the world's least inexpensive areas), then we will work with you on how quickly we reclaim it. Generally this would be over no more than the following 4 months. It is your responsibility if you take this approach to budget appropriately.
```

Comment:
'...and you are travelling **for** over 4 months'

'...from one of the world's most expensive **areas** to...'

Reply:
No comments on the actual approach itself, I think it makes sense.

Reply:
Fixed